master/internal/api_experiment.go

```diff
@@ -499,9 +499,7 @@ func (a *apiServer) deleteExperiments(exps []*model.Experiment, userModel *model
 			}
 
 			// delete jobs per experiment
-			resp, err := a.m.rm.DeleteJob(sproto.DeleteJob{
-				JobID: exp.JobID,
-			})
+			resp, err := a.m.rm.DeleteJob(exp.ResourceManager, exp.JobID)
```

Comment:
don't bother with this until with do slurm multi-rm. (talked offline about detail)